# Available commands to use the Fisher PPS4102 Top Pan balance

int32 ZERO = 0
int32 BALANCE_ON = 1
int32 BALANCE_OFF = 2
int32 MASS_STABLE = 3
int32 MASS = 4

int32 balance_command
